defaults/main.yml

```diff
@@ -3,11 +3,18 @@
 
 mariadb_databases: []
 mariadb_users: []
-mariadb_root_password: ''
+mariadb_root_password: root
 
 mariadb_mirror: yum.mariadb.org
 mariadb_version: '10.4'
 
+mariadb_apt_mirror: 'http://mirror.vpsfree.cz'
+mariadb_apt_repositories: "deb [arch=amd64,arm64,ppc64el] {{ mariadb_apt_mirror }}/mariadb/repo/{{ mariadb_version }}/{{ ansible_distribution | lower }} {{ ansible_distribution_release }} main"
```

Comment:
Since these variables are specific to Apt, I think these should go into `vars/ubuntu.yml`

Reply:
You right! :-)